event.go

```diff
@@ -16,14 +16,81 @@ type Event interface {
 	Revision() *CommitRevision
 	// Validate returns an error if the event is malformed
 	Validate() error
+	// GetProvider returns the name of the provider that created this event
+	GetProvider() string
+	// GetOrganizationID returns the organization to which this event's repository
+	// belongs to
+	GetOrganizationID() string
 }
 
+// EventID is a unique hash id for an event
 type EventID = pb.EventID
+
+// EventType defines the supported event types
 type EventType = pb.EventType
 
+// CommitRevision defines a range of commits, from a base to a head
 type CommitRevision = pb.CommitRevision
+
+// RepositoryInfo contains information about a repository
 type RepositoryInfo = pb.RepositoryInfo
+
+// ReferencePointer is a pointer to a git refererence in a repository
 type ReferencePointer = pb.ReferencePointer
 
-type PushEvent = pb.PushEvent
-type ReviewEvent = pb.ReviewEvent
+// PushEvent represents a Push to a git repository. It wraps the pb.PushEvent
+// adding information only relevant to lookout, and not for the analyzers.
+type PushEvent struct {
+	pb.PushEvent
+	// OrganizationID is the organization to which this event's repository
+	// belongs to
+	OrganizationID string
```

Comment:
I think it should be recorded the `Provider` causing the event, same way we do for `Organization`

Reply:
it's already included in `pb.PushEvent`, https://github.com/src-d/lookout/blob/73ede45df4f6f548090c96bed5b88bf2a8cf0fa9/vendor/gopkg.in/src-d/lookout-sdk.v0/pb/event.pb.go#L118